image_transport/include/image_transport/subscriber_filter.hpp

```diff
@@ -105,13 +105,34 @@ class SubscriberFilter : public message_filters::SimpleFilter<sensor_msgs::msg::
    * \param nh The ros::NodeHandle to use to subscribe.
    * \param base_topic The topic to subscribe to.
    */
+  [[deprecated("Use subscribe(rclcpp::node_interfaces...) instead")]]
```

Comment:
Deprecation message update
```suggestion
  [[deprecated("Use subscribe(..., rclcpp::QoS, ...) instead")]]
```